test/preflight/fly_launch_test.go

```diff
@@ -0,0 +1,222 @@
+//go:build integration
+// +build integration
+
+package preflight
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/superfly/flyctl/test/preflight/testlib"
+)
+
+// TODO: list of things to test
+// - sourceInfo clone vs launch modes
+// - sourceInfo remote dockerfile vs local
+// END
```

Comment:
still relevant?

Reply:
Ohh. those are things done by scanning code that I'd like to test not that I changed them. 

There are lot more flags and even frameworks I'd like to add flighttesting for but had to draw a line to ship this changes.